build.gradle

```diff
@@ -2,13 +2,19 @@ buildscript {
     repositories {
         google()
         mavenCentral()
+        maven { url 'https://developer.huawei.com/repo/' }
```

Comment:
Might be good to enforce only the `com.huawei.*` dependencies resolve on this maven repo?

Reply:
done